stdlib/tkinter/simpledialog.pyi

```diff
@@ -22,6 +25,30 @@ class SimpleDialog:
     def wm_delete_window(self) -> None: ...
     def done(self, num: int) -> None: ...
 
-def askfloat(title: str | None, prompt: str, **kwargs: Any) -> float | None: ...
-def askinteger(title: str | None, prompt: str, **kwargs: Any) -> int | None: ...
-def askstring(title: str | None, prompt: str, **kwargs: Any) -> str | None: ...
+def askfloat(
+    title: str | None,
+    prompt: str,
+    *,
+    initialvalue: float | None = ...,
+    minvalue: float | None = ...,
+    maxvalue: float | None = ...,
+    parent: Misc | None = ...,
+) -> float | None: ...
+def askinteger(
+    title: str | None,
+    prompt: str,
+    *,
+    initialvalue: int | None = ...,
+    minvalue: float | None = ...,  # Any object that can compare to int.
+    maxvalue: float | None = ...,
+    parent: Misc | None = ...,
+) -> int | None: ...
+def askstring(
+    title: str | None,
+    prompt: str,
+    *,
+    initialvalue: str | None = ...,
+    minvalue: str | None = ...,
+    maxvalue: str | None = ...,
```

Comment:
- Why would you want a string to be "between" two other strings? That doesn't make sense to me. I think passing in `minvalue` or `maxvalue` to `askstring()` is more likely a bug than what you wanted to achieve.
- There's a `"show"` keyword argument: `show="*"` would be useful for asking a password, for example.

Reply:
I included min/maxvalue for completeness since they do work (implemented in the common dialog class), but it is indeed fairly useless.